mlrun/serving/routers.py

```diff
@@ -1204,8 +1208,8 @@ def __init__(
         url_prefix: str = None,
         health_prefix: str = None,
         vote_type: str = None,
-        executor_type=None,
-        prediction_col_name=None,
+        executor_type: ParallelRunnerModes = None,
```

Comment:
Change the None to the default value

```suggestion
        executor_type: Union[ParallelRunnerModes, str] = None,
```